doc/modules/model_evaluation.rst

```diff
@@ -636,7 +636,14 @@ by varying a decision threshold.
 The :func:`average_precision_score` function computes the average precision
 (AP) from prediction scores. This score corresponds to the area under the
 precision-recall curve. The value is between 0 and 1 and higher is better.
-With random predictions, the AP is the fraction of positive samples.
+With random predictions, the AP is the fraction of positive samples. AP is
```

Comment:
I support these changes and will make the revisions.

If we remove the AUCPR statement here, I recommend that somewhere we state what the canonical AUCPR calculation is in scikit-learn.  A user who wants to report AUCPR should be able to quickly find the appropriate function.  @amueller do you suggest that the `auc` function on the precision recall operating points is the best option?